test/Container/ApplicationFactoryTest.php

```diff
@@ -65,6 +65,60 @@ public function getRouterFromApplication(Application $app)
         return $r->getValue($app);
     }
 
+    public function testFactorySetsPhpSettingsFromConfig()
+    {
+        $this->container
+            ->has('Zend\Expressive\Router\RouterInterface')
+            ->willReturn(false);
+        $this->container
```

Comment:
This is related with the previous statement. https://github.com/zendframework/zend-expressive/pull/193/files#r45551040

Ok, it must be as you wrote.